libs/SalesforceSDKCore/SalesforceSDKCoreTests/EncryptionTests.swift

```diff
@@ -48,35 +46,21 @@ class EncryptionTests: XCTestCase {
         
         let decryptedData = try Encryptor.decrypt(data: encryptedData, using: keyAgain)
         let decryptedString = String(data: decryptedData, encoding: .utf8)
-        
         XCTAssertEqual(decryptedString, sensitiveInfo)
     }
     
     func testEncryptDecryptWrongKey() throws {
         let key = try KeyGenerator.encryptionKey(for: "test1")
-        XCTAssertNotNil(key)
```

Comment:
I don't think this is needed because `encryptionKey(for:)` method does not return an optional.